<!DOCTYPE html>
  <head>
    <title>Wang Lab</title>
    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
    <style>
      body {
        font-family: 'Open Sans', sans-serif;
      }
    </style>
  </head>
  <body>

    <table>
    <tr><td>
    <img style="margin-right: 20px" src="img/unc_pixel_logo.png" alt="" width="55" height="75" class="alignnone size-full wp-image-86" />
    </td><td>
    Jeremy Wang, Ph.D.
    <br/>
    Department of Genetics
    <br/>
    <span style="font-family:monospace">jeremy &lt;at&gt; TLD</span>
    </td></tr>
    </table>

    <h2>Research</h2>
    My research focuses on metagenomics and long-read (single-molecule) informatics. We develop novel methods to enable more efficient *omic analysis and apply carefully architected high-performance computing approaches to improve the utility of genomics in studies of human disease.

    <h3>Projects</h3>

    <h4>Long-read metagenomics, selective nanopore sequencing, and mucosa-associated gut microbiota in inflammatory bowel disease</h4>
    <p>
    Single-molecule sequencing of metagenomes produces more specific taxonomic and genic characterization than standard NGS, but challenges remain due to relatively low throughput and high error rate, particularly from host-associated microbial communities. We have developed novel sequencing and informatics approaches to increase throughput and classification accuracy in both synthetic and human tissue-associated microbiota.
    </p>

    <h4>Hybrid <i>de novo</i> genome assembly</h4>
    <p>
    Assembly combines next-generation (short) reads with single-molecule (PacBio, Oxford Nanopore) long read sequences. We have developed methods for rapid overlapping and hybrid error-correction of long, error-prone reads. Through several collaborations, we have used these methods and others to produce high-quality assemblies for several bacteria, plants, fungi, <a href="https://www.washingtonpost.com/news/speaking-of-science/wp/2015/11/25/the-invincible-tardigrade-already-a-weird-animal-is-full-of-dna-stolen-from-bacteria/" rel="noopener" target="_blank">water bears</a>, and various diptera.
    </p>

    <h4>Long-read amplicon clustering/denoising</h4>
    <p>
    Various amplicon sequencing-based studies would benefit dramatically from increased read length produced by single-molecule sequencing, including 16S rDNA analysis, HLA typing, and viral evolution experiments. However, deconvolving within-sample variation among full-length amplicons with relatively high error rate is impossible using existing methods. We are developing a method to derive consensus subpopulations via unsupervised clustering able to distinguish unique amplicon populations with sequence divergence far below the sequencing error rate.
    </p>

  </body>
</html>

